simulator/simple_planning_simulator/src/simple_planning_simulator/simple_planning_simulator_core.cpp

```diff
@@ -410,13 +409,13 @@ void SimplePlanningSimulator::add_measurement_noise(
   odom.pose.pose.position.y += (*n.pos_dist_)(*n.rand_engine_);
   const auto velocity_noise = (*n.vel_dist_)(*n.rand_engine_);
   odom.twist.twist.linear.x += velocity_noise;
-  float32_t yaw = motion::motion_common::to_angle(odom.pose.pose.orientation);
+  double yaw = motion::motion_common::to_angle(odom.pose.pose.orientation);
   yaw += static_cast<float>((*n.rpy_dist_)(*n.rand_engine_));
   odom.pose.pose.orientation = motion::motion_common::from_angle(yaw);
 
-  vel.longitudinal_velocity += static_cast<float32_t>(velocity_noise);
+  vel.longitudinal_velocity += static_cast<double>(velocity_noise);
```

Comment:
vel.longitudinal_velocity is defined as a float.
https://github.com/tier4/autoware_auto_msgs/blob/tier4/main/autoware_auto_vehicle_msgs/msg/VelocityReport.idl#L14

Did you replace `float32_t` to `double` on purpose in this PR? Or just mistake?

Reply:
As commented here, on purpose.
I followed the other packages' implementation (= path velocity is handled as double).
https://github.com/autowarefoundation/autoware.universe/pull/2233#discussion_r1015473041